### ISSUE TITLE
Detect browser's language in Safari don't work

### ISSUE THREAD

**Issue by di1606** [isaachinman/next-i18next#921]
## Describe the bug

I set up this package in my projet. However the language detection in the Safari browser does not work. 

My navigator safari and chrome are in French 

When I test this url `http://localhost:3000` in the chrome browser it translates me to this url `http://localhost:3000/fr` which is correct. However when I try it on Safari the url is in English `http://localhost:3000/en`

Where the error may come from ? 

## Version used

- Next.js: v10.0.5
- next-i18next: ^7.0.1
- Safari version: 14.0.2
- Chrome version: 88.0.4324.96

## Steps to reproduce

index.ts
```
import NextI18Next from 'next-i18next'
import path from 'path'
import getConfig from 'next/config'

const { publicRuntimeConfig } = getConfig()
const NextI18NextInstance = new NextI18Next({
  browserLanguageDetection: true,
  defaultLanguage: 'en',
  otherLanguages: ['en', 'fr'],
  localeSubpaths: publicRuntimeConfig.localeSubpaths,
  localePath: path.resolve('./public/translations'),
  defaultNS: 'common'
})
export const { appWithTranslation, useTranslation, withTranslation, Link, i18n} = NextI18NextInstance;
export default NextI18NextInstance;
```

next.config.js

```
const { nextI18NextRewrites } = require('next-i18next/rewrites')

const localeSubpaths = {
  fr: 'fr',
  en: 'en',
}

module.exports = {
  rewrites: async () => nextI18NextRewrites(localeSubpaths),
  publicRuntimeConfig: {
    localeSubpaths,
  },
}
```

_app.tsx

```
function App({ Component, pageProps }: AppProps) {
  return (
    <>
      ...
    </>
  )
}

App.getInitialProps = async (appContext: AppContext) => ({
  ...(await NextApp.getInitialProps(appContext)),
})

export default appWithTranslation(App)
```
index.tsx

```
type Props = {
  readonly t: TFunction
}

function Home({ t }: Props) {
  return (
    <Layout title="Home">
      <h1 className="title">
        {t('home')}
      </h1>
      <h2>{t('overview')}</h2>
      <p className="description">
        Get started by editing <code>pages/index.tsx</code>
      </p>
    </Layout>
  )
}

Home.getInitialProps = async () => ({ namespacesRequired: ['common'] })

export default withTranslation()(Home)
```
## Screenshots
Safari when I load the page `http://localhost:3000`, the cookie generate.
<img width="1440" alt="Capture d’écran 2021-01-26 à 20 31 41" src="https://user-images.githubusercontent.com/27860198/105895233-8d87d080-6015-11eb-94f6-65b857ff7eff.png">

Chrome when I load the page `http://localhost:3000`, the cookie generate.
<img width="1440" alt="Capture d’écran 2021-01-26 à 20 33 57" src="https://user-images.githubusercontent.com/27860198/105895451-d3449900-6015-11eb-97d0-3bfd8e4a0596.png">

**Comment by j4nos** [comment]
Same problem here. Did you figure out solution?

**Comment by isaachinman** [comment]
Hi there, I have just released `next-i18next@8.0.0`, which uses the internationalised routing built into NextJs v10+. This new version of `next-i18next` is quite different, thus making this PR no longer relevant. Thanks!
